legal-api/devops/vaults.gcp.env

```diff
@@ -11,6 +11,7 @@ DATABASE_PASSWORD="op://database/$APP_ENV/business-db/DATABASE_PASSWORD"
 DATABASE_PORT="op://database/$APP_ENV/business-db/DATABASE_PORT"
 DATABASE_UNIX_SOCKET="op://database/$APP_ENV/business-db/DATABASE_UNIX_SOCKET"
 DATABASE_USERNAME="op://database/$APP_ENV/business-db/DATABASE_USERNAME"
+JWT_OIDC_AUDIENCE ="op://keycloak/$APP_ENV/entity-services/ENTITY_SERVICES_CLIENT_ID"
```

Comment:
is this right?  is Patrick going to add this?  I don't see it in 1pass

Reply:
You don't see it as of now right? Going to ask him.

Reply:
yeah, i see no section called "entity-services" under the `keycloak` vault

Reply:
![MicrosoftTeams-image (5)](https://github.com/bcgov/lear/assets/122301442/668c6918-6ea2-4347-be59-47e027fe2c62)

It should be in the keyclock vault according to him

Reply:
ok, figure out where i got confused.  there is a "dev-gcp" category under the `keycloak` vault.  it's not actually being used but i was looking for the config there.  

we should be good here.

Reply:
Deal, sounds good.